packages/create-platformatic/src/create-package-json.mjs

```diff
@@ -7,7 +7,7 @@ const packageJsonTemplate = (addTSBuild = false) => (`\
 {
   "scripts": {
     ${addTSBuild
-? ` "start": "yarn clean && platformatic {type} start",
+? `"start": "yarn clean && platformatic {type} start",
```

Comment:
```suggestion
? `"start": "npm run clean && platformatic {type} start",
```

Let'use use npm here instead

Reply:
The use of `yarn clean` is in `main` too. I created a new TypeScript project with `create-platformatic` yesterday and thought it was odd that the generated `package.json` used `yarn` (which only worked for me as I happen to have `yarn` installed).

I'm happy to open a separate PR to fix this if this PR isn't likely to land for a while.

https://github.com/platformatic/platformatic/blob/main/packages/create-platformatic/src/create-package-json.mjs#L10

Reply:
@simonplend go ahead and do a different PR, it's better we fix this asap.